packages/app/cypress/e2e/runner/support/mochaEventsUtils.ts

```diff
@@ -139,15 +139,18 @@ declare global {
   }
 }
 
-export function runCypressInCypressMochaEventsTest<T> (snapshots: T, snapToCompare: keyof T, done: Mocha.Done) {
+export function runCypressInCypressMochaEventsTest (snapToCompare: string, done: Mocha.Done) {
   const bus = new EventEmitter()
   const outerRunner = window.top!.window
+  const filename = getCallerFile()
+  let snapshots
+
+  cy.task('readMochaEventSnapshot', { filename }).then((existingSnapshots) => {
```

Comment:
Any reason we couldn't use cy.readFile here?

Reply:
I also create the file if it doesn't exist. I may be able to rework this logic a little bit though and create the file during the `write` which would probably make more sense anyway.

Reply:
So, I can't use `readFile` since it assumes the file exists and fails otherwise.

https://docs.cypress.io/api/commands/task#Read-a-file-that-might-not-exist